plugins/modules/problem.py

```diff
@@ -112,6 +114,17 @@
         documentation on creating problems at
         U(https://docs.servicenow.com/bundle/paris-it-service-management/page/product/problem-management/task/create-a-problem-v2.html).
     type: dict
+  base_api_path:
+    description:
+      - Base API path for the ServiceNow problem state management scripted API.
+      - Used for managing problem state transitions.
+      - Requires I(API for Red Hat Ansible Automation Platform Certified Content Collection) to be installed from the ServiceNow Store.
```

Comment:
This is also true where AAP is not installed. isn't it?

Reply:
We will add a link to ServiceNow Application Store here once it is published.

Reply:
Which part does "this" refer to? The scripted API is used especially to transition problem states, which is not possible with the Table API alone, except for advancing state from new to assess. One could argue though that manipulation of roles and ACLs on ServiceNow instance does enable us to reach other states too through Table API, but nevertheless we encounter issues once state becomes resolved or closed. What should be here improved in the text? @Akasurde

Reply:
Link to the application in the ServiceNow Store has been added to the documentation.